### ISSUE TITLE
Make the LTN Train Stop seamless the vanilla Train Stop

### ISSUE THREAD

**Issue by Dominiquini** [Yousei9/Logistic-Train-Network#288]
**TL;DR**
Make the LTN Train Stop seamless replace the vanilla Train Stop

**What ?**
Make that I could replace the vanilla trains stop with the LTN version, without needed to deconstruct first, like a upgrade for belts, assemble machines, etc.

**Why ?**
Would be easier to use if I could replace the normal train stop for the LTN version! I have some blueprints for tracks that use nornal trains stops and other blueprints for LTN stations that use LTN trains stops. I always needed to deploy the track with the normal stop, manually delete the trains stop and deploy de blueprint with the LTN version...

Thanks.

**Comment by Yousei9** [comment]
Possible only for manual replacement.
Blueprints would require changing the base train stop collision box to no longer collide with LTN IO entities.
I'm not going to change base entity date for something insignificant as this and having only manual replacement working feels inconsistent with game mechanics.
